appveyor.yml

```diff
@@ -0,0 +1,51 @@
+platform:
+  - x86
+  - x64
+
+environment:
+  matrix:
+  # Add here environement variables to control the Travis CI build
+    - CONDA_RECIPE: conda
+      CONDA_VERSION: 2
+      ANACONDA_UPLOAD: openalea
+      ANACONDA_LABEL: unstable
+
+install:
+  - git clone https://github.com/OpenAlea/appveyor-ci.git appveyor-ci
+  - cd appveyor-ci
+  - call install.bat
+  - conda config --add channels openalea/label/unstable
```

Comment:
You remove it in PlantGL. Do you to keep it here?
Add openalea/label/unstable in the settings.